homeassistant/components/rabbitair/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+  "domain": "rabbitair",
+  "name": "Rabbit Air",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/rabbitair",
+  "requirements": [
+    "python-rabbitair==0.0.6"
+  ],
+  "ssdp": [],
+  "zeroconf": [
+    "_rabbitair._udp.local."
+  ],
+  "homekit": {},
+  "after_dependencies": [
+    "zeroconf"
```

Comment:
How is `zeroconf` used that it is needed as an after dependency?

Reply:
An instance of AsyncZeroconf is passed to the library so that it can resolve mDNS names. And, according to the [docs](https://developers.home-assistant.io/docs/network_discovery#mdnszeroconf), in this case `zeroconf` should be added to `dependencies` in the integration's `manifest.json`. But since mDNS functionality is optional, `after_dependencies` is used instead of `dependencies` (and I see the same in other integrations).

It should be noted that the current firmware does not support zeroconf discovery, but does support mDNS. However, this should not be a problem. The next update will add full support for zeroconf.